src/tokenized-line.coffee

```diff
@@ -111,27 +111,32 @@ class TokenizedLine
 
       return maxColumn
 
-  # Calculates how many trailing spaces in this line's indentation cannot fit in a single tab.
+  # For a given `indentLevel`, calculates how many trailing spaces cannot fit in a single tab.
   #
-  # Returns a {Number} representing the odd indentation spaces in this line.
-  getOddIndentationSpaces: ->
-    oddIndentLevel = @indentLevel - Math.floor(@indentLevel)
+  # indentLevel - {Number}
+  # Returns a {Number} representing the odd indentation spaces in `indentLevel`.
+  oddIndentationSpacesForIndentLevel: (indentLevel) ->
+    oddIndentLevel = indentLevel - Math.floor(indentLevel)
```

Comment:
Actually, I have a suggestion for removing this method entirely by changing the implementation of `buildSoftWrapIndentationTokens` below...